### PR TITLE
this commit will add b3NewFile2

### PR DESCRIPTION
**Pull request by Nagendra7509**
this commit will add b3NewFile2

### CODE REVIEW

**Review by RajaTheKing826 — COMMENTED** [review]
why its dp why its not your name

File: b3NewFile2.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+class Durga extends React.Component {
+  render() {
+    return <div>DP</div>;
```
Comment (RajaTheKing826) [inline b3NewFile2.js]: why its dp change it to your name

Comment (Nagendra7509) [reply b3NewFile2.js]: ok sir

File: b3NewFile2.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+class Durga extends React.Component {
```
Comment (prasaduu) [inline b3NewFile2.js:3]: Why the component name is mine & Div contains your name

File: b3NewFile2.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 
 class Durga extends React.Component {
   render() {
-    return <div>DP</div>;
+    return <div>Nagendra</div>;
```
Comment (RajaTheKing826) [inline b3NewFile2.js:5]: nice this is approved now go and enjoy this weekend

Comment (RajaTheKing826) [inline b3NewFile2.js:5]: nice go and enjoy weekend

**Pull request merged**